linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/tests/pam_tally2_deny_missing.fail.sh

```diff
@@ -8,10 +8,24 @@ account required                        pam_tally2.so
 account	required			pam_permit.so
 CAPTAC
 
+{{% if product in ["sle12","sle15"] %}}
```

Comment:
The rule is restricted to sle products. So, why the conditionals here? Would be simpler to only update the file.